x/observer/migrations/v3/migrate.go

```diff
@@ -0,0 +1,32 @@
+package v3
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/observer/keeper"
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+// MigrateStore migrates the x/observer module state from the consensus version 2 to 3
+// This migration update the policy group
+func MigrateStore(ctx sdk.Context, k keeper.Keeper) error {
+	// Get first admin policy group
+	p := k.GetParams(ctx)
```

Comment:
Maybe we can use `GetParamsIsExists` instead of `GetParams` just to be safe .

Reply:
alternatively, we can just use `DefaultAdminPolicy` as we know `GroupID1Address` is owned by us.

Reply:
Switched to `GetParamsIsExists` and added tests